api/models/ticket.py

```diff
@@ -28,7 +28,7 @@ class Ticket(HasUuid):
 
     members_pins = models.ManyToManyField("Member", through="PinnedTicket")
     tag_list = models.ManyToManyField("Tag", through="TicketTag")
-    due_date = models.DateTimeField(null=True, blank=True)
+    due_date = models.DateField(null=True, blank=True)
```

Comment:
Not sure about this one. I think we want to include time with the due date

Reply:
I agree with Sam here, even if we aren't specifically displaying it on one client it's an important feature to have. If we ever want to do ranking on what should be seen in the homepage by due dates it helps to have the backend differentiate by not just date.